frontend/ui/elements/common_settings_menu_table.lua

```diff
@@ -315,7 +315,49 @@ common_settings.document = {
                     end,
                 },
 
-            }
+            }		
+        },        
+	{
+            text = _("Action on highlight"),
+            sub_item_table = {
+                {
+                    text = _("Default"),
+                    checked_func = function()
+                        local setting = G_reader_settings:readSetting("default_highlight_action")
+                        return setting == "ask" or setting == nil
+                    end,
+                    callback = function()
+                        G_reader_settings:saveSetting("default_highlight_action", "ask")
+                    end,
+                },
+                {
+                    text = _("Highlight"),
+                    checked_func = function()
+                        return G_reader_settings:readSetting("default_highlight_action") == "highlight"
+                    end,
+                    callback = function()
+                        G_reader_settings:saveSetting("default_highlight_action", "highlight")
+                    end,
+                },
+                {
+                    text = _("Translate"),
+                    checked_func = function()
+                        return G_reader_settings:readSetting("default_highlight_action") == "translate"
+                    end,
+                    callback = function()
+                        G_reader_settings:saveSetting("default_highlight_action", "translate")
+                    end,
+                },
+                {
+                    text = _("Check wikipedia"),
```

Comment:
```suggestion
                    text = _("Search Wikipedia"),
```

Or just plain `Wikipedia` or `Wikipedia lookup`.